lib/mollie/api/client.rb

```diff
@@ -89,10 +70,9 @@ def perform_http_call(http_method, api_method, id = nil, http_body = {})
             raise Mollie::API::Exception.new("Invalid HTTP Method: #{http_method}")
         end
 
-        request.add_field('Accept-Encoding', 'application/json')
-        request.add_field('Authorization', "Bearer #{@api_key}")
-        request.add_field('User-Agent', @version_strings.join(" "))
-        request.add_field('X-Mollie-Client-Info', uname)
+        request['Accept']        = 'application/json'
+        request['Authorization'] = "Bearer #{@api_key}"
+        request['User-Agent']    = "Mollie-Ruby/#{VERSION}"
```

Comment:
We would like to keep the user agent of the different clients somewhat consistent, something like `Mollie/2.0.0 Ruby/2.3.1 OpenSSL/0.9.8zh`. Is there a reason why you want to change this?